README.md

```diff
@@ -83,10 +83,21 @@ Envless is licensed under the GNU Affero General Public License v3.0 License - s
 <h2 id="contributors">✨ Contributors</h2>
 
 + We ❤️ contributors! Feel free to contribute to this project!. 
++ Any contributions you make are truly appreciated.
++ Let's continue contributing to keep the community active and growing.
 + We also welcome you to join our [Slack](https://dub.sh/envless-slack) or [Discord](https://dub.sh/envless-discord) community for either support or contributing guidance.
 
 <a href="https://github.com/envless/envless/graphs/contributors">
   <p>
     <img src="https://contrib.rocks/image?repo=envless/envless" alt="A table of avatars from the project's contributors" />
   </p>
 </a>
+<h2>🙏 Support</h2>
+<p>
+Don't forget to leave a star ⭐️
+  </p>
+  
+  ![Enveless](https://user-images.githubusercontent.com/111564362/221513732-189c3917-2ef5-48f8-b5d9-334492389764.gif)
```

Comment:
Thanks @kumarankit999 for quick turnaround. Please add this gif to ./github/images folder and link it from there instead. Here is an example

Image: https://github.com/envless/envless/blob/main/.github/images/discord-banner.png